extensions/wikia/Forum/ForumController.class.php

```diff
@@ -194,7 +192,7 @@ public function boardThread() {
 		wfProfileOut( __METHOD__ );
 	}
 
-	public function breadCrumbs() {
+	public function brickHeader() {
 		if ( $this->app->wg->Title->getNamespace() == NS_WIKIA_FORUM_TOPIC_BOARD ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)